src/efi.rs

```diff
@@ -611,23 +611,20 @@ pub(crate) fn create_efi_boot_entry(
     }
     let loader = format!("\\EFI\\{}\\{SHIM}", vendordir);
     log::debug!("Creating new EFI boot entry using '{target}'");
-    let st = Command::new(EFIBOOTMGR)
-        .args([
-            "--create",
-            "--disk",
-            device,
-            "--part",
-            partition_number.as_str(),
-            "--loader",
-            loader.as_str(),
-            "--label",
-            target,
-        ])
-        .status()?;
-    if !st.success() {
-        anyhow::bail!("Failed to invoke {EFIBOOTMGR}")
-    }
-    anyhow::Ok(())
+    let mut cmd = Command::new(EFIBOOTMGR);
+    cmd.args([
+        "--create",
+        "--disk",
+        device,
+        "--part",
+        partition_number.trim(),
```

Comment:
Hum, you changed this from .as_str() to .trim(). Is this expected?

Reply:
When I run this, find that the `partition_number` has `\n`, so I change it to `trim()` to remove the it.

Reply:
Add this in separate commit. Thank you for the review!